src/sentry/static/sentry/app/components/bases/pluginComponentBase.jsx

```diff
@@ -11,7 +11,7 @@ import {
 } from 'app/actionCreators/indicator';
 import {t} from 'app/locale';
 
-const callbackWithArgs = function(callback, ...args) {
+const callbackWithArgs = function (callback, ...args) {
```

Comment:
Please, should we really introduce the " space-before-function-parentheses" rule?

Reply:
This wasn't added by an eslint rule, it is coming from prettier which doesn't offer a configuration point for this.